cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -808,6 +815,42 @@ impl fmt::Display for CC {
     }
 }
 
+/// Encode the ways that floats can be compared.
+pub(crate) enum FloatComparisonPredicate {
```

Comment:
I'm open to a different name but it should probably have the `Float` in the name somewhere and I'm not too sure about `Mask`--it relates to the instruction semantics but not directly to the predicate so that is a bit confusing.